tests/test_coastal_vulnerability.py

```diff
@@ -713,26 +713,67 @@ def test_complete_run(self):
         args['slr_field'] = 'Trend'
         coastal_vulnerability.execute(args)
 
-        actual_values_df = pandas.read_csv(
-            os.path.join(args['workspace_dir'], 'coastal_exposure.csv'))
-        expected_values_df = pandas.read_csv(
-            os.path.join(REGRESSION_DATA, 'expected_coastal_exposure.csv'))
-        pandas.testing.assert_frame_equal(
-            actual_values_df, expected_values_df, check_dtype=False)
-
-        # Also expect matching shore_id field in all tabular outputs:
-        intermediate_csv = pandas.read_csv(
-            os.path.join(
-                args['workspace_dir'],
-                'intermediate/intermediate_exposure.csv'))
-        habitat_csv = pandas.read_csv(
-            os.path.join(
-                args['workspace_dir'],
-                'intermediate/habitats/habitat_protection.csv'))
-        pandas.testing.assert_series_equal(
-            actual_values_df['shore_id'], intermediate_csv['shore_id'])
-        pandas.testing.assert_series_equal(
-            actual_values_df['shore_id'], habitat_csv['shore_id'])
+        results_vector = gdal.OpenEx(
+            os.path.join(args['workspace_dir'], 'coastal_exposure.gpkg'),
+            gdal.OF_VECTOR)
+        results_layer = results_vector.GetLayer()
+
+        # shore points aren't necessarily created/numbered in the same order,
+        # so we can't compare by the fid or shore_id fields

```

Comment:
I think we *can* expect the points be created/numbered in the same order each time the same version of the model runs. I'm guessing something upstream (shapely?) changed to make the order change between versions.

Reply:
The shapely version was the same (1.7.1) between passing and failing runs. I expect they would be created in the same order each time, but I don't think shapely guarantees it. They just describe a MultiPolygon as a "collection".